### DISCUSSION TITLE
Could it be as it appears that ungoogled-chromium lacks auto-update support?

### DISCUSSION THREAD

**Discussion by Ricky-Tigg** [Eloston/ungoogled-chromium#2740] [Q&A]
Good morning. I was able to draw from my own experience the conclusion that this browser in stable version for Windows 64-bit (tested: variant 11, version 23H2) lacks auto-update support. I was able to see that this was also the case with the Chromium code, for Windows 64-bit, through an acquisition at the source https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Win_x64/ of a portable type archive, for example that for the _1267524_-build corresponds to the relative path 1267524/_chrome-win.zip_. However, the files from this source represent without exception developer type versions, which are here at https://www.chromium.org/getting-involved/download-chromium/ explicitly presented as being devoid of auto-updating support.

This browser, although presented as close to Chromium in terms of code, is nonetheless unique in its differences. Anyone who, like me, discovers this browser, can reasonably assume that auto-update support has been introduced, this being of undeniable convenience. However at https://ungoogled-software.github.io/ungoogled-chromium-wiki/faq, which is the place where we are entitled to see a question asked relating to this support, we surprisingly see its absence. Neither does the _README.md_ file mention the introduction of such support in this browser.

Could it be that such support exists but is disabled by a flag? It seems inconceivable these days to be put in the position of having to watch for the release of a new version, for example via the built-in function of this platform known as _Watch_, in order to manually acquire it, and proceed with its extraction or installation.

**Comment by PF4Public** [comment]
>Anyone […] can reasonably assume that auto-update support has been introduced

Incorrect

This feature is very platform dependent and on Windows ungoogled-chromium follows  Chromium closely. On Linux however this feature is often implemented by a package manager instead.


> having to watch for the release of a new version, for example via the built-in function of this platform known as _Watch_, in order to manually acquire it, and proceed with its extraction or installation.

There is no such functionality in ungoogled-chromium, but fortunately @claudiodekker created https://github.com/claudiodekker/chromium-update-notifier which might be fitting for your needs. I haven't used it so cannot suggest or give advices on it. Also I should mention that it is developed independently from ungoogled-chromium and in no way related to it.
